guide/src/advanced/troubleshooting/clock-drift.md

```diff
@@ -0,0 +1,32 @@
+# Handling Clock Drift
+
+When it comes to distributed systems, clock drift is always a consideration that needs to be 
+taken into account. There is always some amount of drift between the clock of the source chain 
+and the clock of the destination chain that must be accounted for. As far as Hermes is concerned,
+there are two configuration parameters that exist to specify how much clock drift ought to be tolerated: 
+ - `clock_drift` is a correction parameter that specifies how far in the future or past a chain's clock may
+   drift in relation to this client. The destination chain for this client uses this parameter when deciding
+	 whether to accept or reject a new header from the source chain for this client.
+ - `max_block_time` is the maximum amount of time that can occur before a new block is created on the chain. 
+
+> **Note:** When it comes to Cosmos SDK chains, a good approximation for `max_block_time` is
+`timeout_propose` + `timeout_commit`, which together sum up to the total amount of time before
+the proposal and commitment steps for a new block time out. 
```

Comment:
```suggestion
> **Note:** For Cosmos SDK chains, a good approximation for `max_block_time` is
`C * (timeout_propose + timeout_commit)`. To allow for some variance in block times while still detecting forward lunatic attacks (todo - add reference), it is recommended to set `C` to be in the `3..5` range. Hermes uses the default value of `30s` which is a good approximation for most Cosmos SDK chains that have 6-10sec block times.
```

Reply:
It's a constant to allow for variation in block times, mainly due to Tx execution time which is outside of consensus params.